pysmt/test/smtlib/test_omt_lib_solver.py

```diff
@@ -329,5 +334,29 @@ def test_omt_lex_unsat(self):
         )
 
 
+# define a dummy to have access to the assertEqual etc. methods
+# inside the parametrized tests
+dummy = TestCase()
```

Comment:
this is a bit weird. Is there no other solution?

Reply:
I did a bit more research and didn't find a better option.

This is to be able to use the `assertEqual(x1, x2)` etc. methods, which are better than `assert x1 == x2` and provide a clearer error message.

The dummy could also be created inside the single parametrized test that uses it.

Let me know which of the 3 options you like more:
1) leave as is
2) move the dummy creation inside the parametrized test
3) use the simple python `assert` instead of the `unittest` methods

Reply:
It seems that solution 3 is the one suggested on the official documentation and is the one we also use for the other smtlib tests, so let's go with that